docker/Dockerfile-api

```diff
@@ -9,31 +9,18 @@ RUN dnf -y install \
     httpd \
     mod_auth_gssapi \
     mod_ssl \
-    python3-celery \
-    python3-flask \
-    python3-flask-login \
-    python3-flask-migrate \
-    python3-flask-sqlalchemy \
-    python3-GitPython \
-    python3-mod_wsgi \
-    python3-packaging \
     python3-pip \
-    python3-psycopg2 \
-    python3-requests \
-    python3-requests-kerberos \
-    python3-semver \
-    python3-setuptools \
+    python3-mod_wsgi \
     && dnf clean all
 COPY . .
 
-# FIXME: there was a regression in py-amqp. It was supposed to be fixed in
-# https://github.com/celery/py-amqp/pull/350
-# but cachito is still being affected by the regression
-RUN rpm -e --nodeps python3-amqp && pip3 install amqp==5.0.2
+RUN rpm -e --nodeps python3-amqp
 
 COPY ./docker/cachito-httpd.conf /etc/httpd/conf/httpd.conf
-# All the requirements except pyarn should already be installed
-RUN pip3 install pyarn && \
-    pip3 install . --no-deps
+
+RUN pip3 install -r requirements.txt --no-deps --require-hashes
+RUN pip3 install -r requirements-web.txt --no-deps --require-hashes
+RUN pip3 install . --no-deps
```

Comment:
Usually, people combine multiple RUN instructions into one like this:
```Dockerfile
RUN a && \
    b && \
    c
```
I think the reason for that is that you reduce the number of layers that way. I don't know if that's really a best practice though, what do you think?

CC  @athos-ribeiro @lcarva

Reply:
yes, you are right. I just saw it in [iib Dockerfile-api](https://github.com/release-engineering/iib/blob/master/docker/Dockerfile-api) Let's wait for @lcarva :D 

Maybe it worth to keep it as a different layer.

Reply:
Lui answered that it will not add additional files for the layer, so we can keep it in both ways.

Reply:
We can keep it as it is :+1: